### ISSUE TITLE
Black screen after booting 

### ISSUE THREAD

**Issue by unicornsandcandies** [ublue-os/bazzite#1159]
### Describe the bug

Sometimes when booting, my device boots to black screen. I've managed to ssh and get journal from current boot.

[journal.log](https://github.com/ublue-os/bazzite/files/15393459/journal.log)

### What did you expect to happen?

I expected bazzite to start game scope session

### Output of `rpm-ostree status`

```shell
State: busy
Transaction: upgrade
  Initiator: client(id:cli dbus:1.146 unit:ublue-update-force.service uid:0)
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:3cb3d26b07ebb40a45b62b89650db1048d0b88eb139d1acbc4b737b29d2182e8
                  Version: 40.20240520.0 (2024-05-21T01:03:06Z)
          LayeredPackages: plymouth-theme-script
            LocalPackages: pedro-raccoon-1.1-1.1.x86_64
                Initramfs: regenerate

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:571e1c2fb00822e12a88ea0dabb631074c88356ab5213a829a7c178f3d286bfe
                  Version: 40.20240517.0 (2024-05-17T17:09:39Z)
          LayeredPackages: plymouth-theme-script
            LocalPackages: pedro-raccoon-1.1-1.1.x86_64
                Initramfs: regenerate
```

### Hardware

Steam Deck OLED

**Comment by wolfyreload** [comment]
I don't have a handheld but I've seen a number of people have success with the 40-20240427 image. Could you try a `bazzite-rollback-helper rebase 40-20240427` and see if that works

**Comment by unicornsandcandies** [comment]
> I don't have a handheld but I've seen a number of people have success with the 40-20240427 image. Could you try a `bazzite-rollback-helper rebase 40-20240427` and see if that works

It seems to help. I don't really want to use old versions of bazzite, so I hope that this will be fixed soon

**Comment by wolfyreload** [comment]
> > I don't have a handheld but I've seen a number of people have success with the 40-20240427 image. Could you try a `bazzite-rollback-helper rebase 40-20240427` and see if that works
> 
> It seems to help. I don't really want to use old versions of bazzite, so I hope that this will be fixed soon

Agreed. It might be worth pinning the working version with `sudo ostree admin pin 0` and just try the new updates once a week. I hope that the issue is fixed soon.

**Comment by nmlynch94** [comment]
Same issue here on steam OLED. Rolling back worked, but I had to run the command on first boot after installation. Otherwise, after the first reboot, dropping to a TTY just ended up being unresponsive.

**Comment by hhawkens** [comment]
Same here with my Home Theater System. I cannot update for more than a month now.

**Comment by hhawkens** [comment]
Update: It appears to be fixed for me with image **611.0**
